build-docker.sh

```diff
@@ -17,7 +17,7 @@ key="$1"
 case $key in
     --gpu)
     GPU="-gpu"
-    BASE_IMAGE="nvidia/cuda:11.0-cudnn8-runtime-ubuntu18.04"
+    BASE_IMAGE="nvidia/cuda:11.3-cudnn8-devel-ubuntu18.04"
```

Comment:
Does [Tensorflow](https://www.tensorflow.org/install/gpu) work with Cuda 11.1?

Reply:
On Horovod, we've been successfully running TF built with CUDA 11.0 and PyTorch built with CUDA 11.1 using CUDA 11.2 image in our CI process: https://github.com/horovod/horovod/blob/master/docker-compose.test.yml#L200

Reply:
This should probably be tested thoroughly, though, so to unblock I will change back to 11.0.